InputOutput.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace cookBook
+{ 
+    internal class InputOutput
+    {
+        static RecipeManager recipeBook = new RecipeManager();
```

Comment:
Bardzo spoko, dobrze to podzieliłeś.
Następny krok to na ten moment zrobić te poprawki co opisałem.
Następny PR (albo ten jeśli chcesz) to wprowadzenie interfejsów dla wszystkich klas których stworzyłeś.
Gdzie się da nie chcielibyśmy zależności klasy od klasy (pomijając klas-typów).
W dzisiejszym programowaniu memberami klas są interfejsy.
Ta linijka powinna wyglądać IRecipeManager recipeBook = new RecipeManager().
Później wyewoluje bardziej.
Nie chcemy mieć tutaj singletona, static nam niepotrzebny.
Bardzo rzadko będziemy używać staticów, czynią one kod nietestowalnym (o tym później)